pymc3/distributions/distribution.py

```diff
@@ -355,7 +357,9 @@ def _draw_value(param, point=None, givens=None):
         if point and hasattr(param, 'model') and param.name in point:
             return point[param.name]
         elif hasattr(param, 'random') and param.random is not None:
-            return param.random(point=point, size=None)
+            return param.random(point=point, size=None).mean(axis=0)
+        elif hasattr(param, 'random') and param.random is not None and size is not None:
```

Comment:
this branch will never get hit (since the above condition will always hit)

Reply:
Ok I'll remove that branch.